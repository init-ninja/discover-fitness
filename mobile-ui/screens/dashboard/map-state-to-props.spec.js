import mapStateToProps from './map-state-to-props'

const state = {
  app: {
    user: {
      isLoggedIn: true,
      firstName: 'FirstName',
      lastName: 'LastName',
      profilePictureUrl: 'profile.picture.url'
    }
  }
}

describe('screens > dashboard > map-state-to-props', () => {
  it('should behave OK', () => {
    expect(mapStateToProps(state)).toMatchSnapshot()
  })
})
