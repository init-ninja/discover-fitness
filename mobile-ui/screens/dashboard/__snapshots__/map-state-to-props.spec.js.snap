// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screens > dashboard > map-state-to-props should behave OK 1`] = `
Object {
  "user": Object {
    "displayName": "FirstName",
    "isLoggedIn": true,
    "profilePictureUrl": "profile.picture.url",
  },
}
`;
